generator/resources/ec2_acceptance_test_matrix.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "os": "ubuntu-20.04",
+    "username": "ubuntu",
+    "instanceType":"t3a.medium",
+    "installAgentCommand": "go run ./install/install_agent.go deb",
+    "ami": "cloudwatch-agent-integration-test-ubuntu*",
+    "caCertPath": "/etc/ssl/certs/ca-certificates.crt",
+    "arc": "amd64",
```

Comment:
Typically the abbreviation is "arch", I do not expect you to change it in this PR though.